aip/general/0157.md

```diff
@@ -78,10 +78,36 @@ enum BookView {
 
 ### Read masks as a request field
 
-**Warning:** Read masks as a single field on the request message, for example: `google.protobuf.FieldMask read_mask` are **DEPRECATED**.
+**Warning:** Read mask as a single, explicit field on the request message is
+**DEPRECATED** for Google APIs. The [system parameter](#field-masks-parameter)
+**must** be used instead. The following guidance is for the benefit of existing
```

Comment:
legacy? historical? grandfathered?

Reply:
Changed to `existing legacy Google and external non-Google usage`

Is that what you wanted to see? PTAL!

Reply:
Yep!